luacompiler/src/lib/irgen/opcodes.rs

```diff
@@ -0,0 +1,23 @@
+use bytecode::instructions::Opcode;
+
+#[derive(Debug, Copy, Clone, PartialEq)]
+pub enum IROpcode {
+    Opcode(Opcode),
+    Phi,
```

Comment:
Naively, I was expecting this to have one or more parents node pointers?

Reply:
The basic blocks record their parents, and phis are generated based on that.

Reply:
OK.